### PR TITLE
Fixed wizard slaughter demons not getting bloodcrawl

### PR DESCRIPTION
**Pull request by xxalpha**
Fixes https://github.com/tgstation/-tg-station/issues/11540

### CODE REVIEW

File: code/modules/mob/living/simple_animal/slaughter/slaughter.dm [hunk 1]
```diff
@@ -36,6 +36,15 @@
 						Pulling a dead or critical mob while you enter a pool will pull them in with you, allowing you to feast. \
 						You move quickly upon leaving a pool of blood, but the material world will soon sap your strength and leave you sluggish. </B>"
 
+/mob/living/simple_animal/slaughter/New()
+	. = ..()
+	spawn(15) //wait for a mind to take over
+		if(mind)
```
Comment (Jordie0608) [inline code/modules/mob/living/simple_animal/slaughter/slaughter.dm]: Don't check for mind, add it to the mob's spell list.

**Comment by Jordie0608** [comment]
You'll need to remove the spell being added when a wizard spawns one too.

**Comment by Xhuis** [comment]
this probably needs to be speedmerged

**Comment by xxalpha** [comment]
Nah.

**Comment by Anonus** [comment]
GOODJOB

**Pull request merged**